ui/apps/platform/src/Containers/Integrations/IntegrationForm/Forms/CloudSourceIntegrations/OcmIntegrationForm.tsx

```diff
@@ -0,0 +1,422 @@
+                                            . The service account must belong to a group with the{' '}
+                                            <strong>Subscription Viewer</strong> role. See{' '}
+                                            <Button
+                                                variant="link"
+                                                isInline
+                                                component="a"
+                                                href="https://docs.redhat.com/en/documentation/openshift_cluster_manager/1-latest/html-single/managing_clusters/index#assembly-user-management-ocm"
+                                                target="_blank"
+                                                rel="noreferrer"
+                                            >
+                                                Configuring access to clusters in OpenShift Cluster
+                                                Manager
+                                            </Button>{' '}
+                                            for more information.
+                                        </FlexItem>
+                                        <FlexItem>
+                                            <em>
+                                                Service accounts are the preferred authentication
+                                                method over the deprecated API token.
+                                            </em>
+                                        </FlexItem>
+                                    </Flex>
+                                }
+                                ariaLabel="Help for client ID"
+                            />
+                        }
+                        fieldId="cloudSource.credentials.clientId"
+                        touched={touched}
+                        errors={errors}
+                    >
+                        <TextInput
+                            isRequired={
+                                values.updateCredentials && !values.cloudSource.credentials.secret
+                            }
+                            type="text"
+                            id={`cloudSource.credentials.clientId`}
+                            value={values.cloudSource.credentials.clientId}
+                            onChange={(event, value) => onChange(value, event)}
+                            onBlur={handleBlur}
+                            isDisabled={!isEditable || !values.updateCredentials}
+                            placeholder={
+                                values.updateCredentials
+                                    ? ''
+                                    : 'Currently-stored client ID will be used.'
+                            }
+                        />
+                    </FormLabelGroup>
+                    <FormLabelGroup
+                        isRequired={values.updateCredentials}
+                        label="Client secret"
```

Comment:
Ditto the previous comment:
![Client_secret](https://github.com/user-attachments/assets/ec030b3d-830a-4db3-bf30-c9af14a2385a)

```tsx
                        label="Client secret"
                        labelIcon={
                            <IntegrationHelpIcon
                                hasAutoWidth
                                helpTitle="Service account client secret"
                                helpText={
                                    <Flex direction={{ default: 'column' }}>
                                        <FlexItem>
                                            Client secret for a{' '}
                                            <ExternalLink>
                                                <a
                                                    href="https://console.redhat.com/iam/service-accounts"
                                                    target="_blank"
                                                    rel="noopener noreferrer"
                                                >
                                                    Red Hat service account
                                                </a>
                                            </ExternalLink>
                                        </FlexItem>
                                        <FlexItem>
                                            The service account must belong to a group with the{' '}
                                            <strong>Subscription Viewer</strong> role.
                                        </FlexItem>
                                        <FlexItem>
                                            For more information, see{' '}
                                            <ExternalLink>
                                                <a
                                                    href="https://docs.redhat.com/en/documentation/openshift_cluster_manager/1-latest/html-single/managing_clusters/index#assembly-user-management-ocm"
                                                    target="_blank"
                                                    rel="noopener noreferrer"
                                                >
                                                    Configuring access to clusters in OpenShift
                                                    Cluster Manager
                                                </a>
                                            </ExternalLink>
                                        </FlexItem>
                                        <FlexItem>
                                            <em>
                                                Service accounts are the preferred authentication
                                                method over the deprecated API token.
                                            </em>
                                        </FlexItem>
                                    </Flex>
                                }
                                ariaLabel="Help for client secret"
                            />
                        }
```

Reply:
done, see [2754468](https://github.com/stackrox/stackrox/pull/12277/commits/27544685b601615d3fd113fbc08622e40c1599e3)